src/Sulu/Bundle/PageBundle/Resources/js/views/Form/toolbarActions/TemplateToolbarAction.js

```diff
@@ -30,6 +31,20 @@ export default class TemplateToolbarAction extends AbstractFormToolbarAction {
             throw new Error('The ToolbarAction for types only works with entities actually supporting types!');
         }
 
+        const {
+            visible_condition: visibleCondition,
```

Comment:
This `ToolbarAction` is only usable in the `PageAdmin`, and there we don't need this property, right? Then I would not implement it as well.